tests/phpunit/tests/formatting/LikeEscape.php

```diff
@@ -10,21 +10,21 @@ class Tests_Formatting_LikeEscape extends WP_UnitTestCase {
 	 */
 	function test_like_escape() {
 
-		$inputs = array(
+		$inputs   = array(
 			'howdy%', //Single Percent
 			'howdy_', //Single Underscore
 			'howdy\\', //Single slash
 			'howdy\\howdy%howdy_', //The works
 		);
 		$expected = array(
-			"howdy\\%",
+			'howdy\\%',
```

Comment:
Single quotes in use upstream:
https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/formatting/likeEscape.php

Reply:
This seems wrong. Even the inputs to that test don't match the comment, so I can see how the test would pass because the quotes changed.
The whole test needs fixing.

Reply:
I think I was misreading it. I tried a little test to prove to myself what it's doing.

Reply:
It turns out that this test is for a deprecated function.